libbeat/docs/newbeat.asciidoc

```diff
@@ -92,32 +99,33 @@ The following example shows an event object in Lsbeat:
 
 Now that you have the big picture, let's dig into the code.
 
-=== Generate your Beat
+=== Generating Your Beat
+
+To generate your own Beat, you use the Beat generator available in the beats repo on GitHub. If you haven't
+downloaded the Beats source code yet, follow the instructions in <<newbeat-getting-ready>>.
 
-To generate your own beat we use the `beat-generator`. First you must install
-https://github.com/audreyr/cookiecutter[cookiecutter]. Check out the
-http://cookiecutter.readthedocs.io/en/latest/installation.html[installation
-guide]. After having installed cookiecutter, we must decide on a name for our beat. 
-The name must be one word all lowercase. In our example is Testbeat.
+Before running the Beat generator, you must install https://github.com/audreyr/cookiecutter[cookiecutter], a
+command-line utility that creates projects from project templates. Check out the 
+http://cookiecutter.readthedocs.io/en/latest/installation.html[installation guide]. After installing cookiecutter,
+decide on a name for your beat. The name must be one word in all lowercase. In our example, we use `testbeat`.
```

Comment:
@dedemorton Oh. My mistake here. I updated only the example, but not the text. The way we write the Beat name is starting with a capital later, and then the rest lowercase, and keep all lowercases for libbeat. I am thinking that it would be nice to advice our users to do the same.